<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-O4BFA6G" failures="2" tests="3" name="ErrorTest" time="21.402" errors="0" timestamp="2024-06-01T21:05:58 IST">
  <testcase classname="testcases.ErrorValidationgroups" name="ErrorTest" time="0.717">
    <skipped/>
  </testcase> <!-- ErrorTest -->
  <testcase classname="testcases.ErrorValidationgroups" name="ErrorTest" time="1.666">
    <failure type="java.lang.AssertionError" message="expected [Incorrect email or password.##] but found [Incorrect email or password.]">
      <![CDATA[java.lang.AssertionError: expected [Incorrect email or password.##] but found [Incorrect email or password.]
at testcases.ErrorValidationgroups.ErrorTest(ErrorValidationgroups.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- ErrorTest -->
  <testcase classname="testcases.ErrorValidationgroups" name="ProductErrorValidation" time="3.343">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testcases.ErrorValidationgroups.ProductErrorValidation(ErrorValidationgroups.java:36)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- ProductErrorValidation -->
</testsuite> <!-- ErrorTest -->
